homeassistant/components/webostv/quality_scale.yaml

```diff
@@ -0,0 +1,61 @@
+rules:
+  # Bronze
+  action-setup: todo
+  appropriate-polling: todo
+  brands: done
+  common-modules:
+    status: exempt
+    comment: The integration does not use common patterns.
+  config-flow-test-coverage: todo
+  config-flow: todo
+  dependency-transparency: todo
+  docs-actions: todo
+  docs-high-level-description: todo
+  docs-installation-instructions: todo
+  docs-removal-instructions: todo
+  entity-event-setup: todo
+  entity-unique-id: todo
+  has-entity-name: todo
+  runtime-data: todo
+  test-before-configure: todo
+  test-before-setup: todo
+  unique-config-entry: todo
+
+  # Silver
+  action-exceptions: todo
+  config-entry-unloading: todo
+  docs-configuration-parameters: todo
+  docs-installation-parameters: todo
+  entity-unavailable: todo
+  integration-owner: todo
+  log-when-unavailable: todo
+  parallel-updates: todo
+  reauthentication-flow: todo
+  test-coverage: todo
```

Comment:
`test_client_disconnected` feels like it is only touching code. What would we otherwise expect to happen?

Reply:
When TV is off we suppress some errors so they are not shown to the user:
https://github.com/home-assistant/core/blob/4b6febc7579e113f8c22dbf64ff09193fde8445e/homeassistant/components/webostv/media_player.py#L328

I think this is still a valid test to make sure this code block is not changed by someone in the future

Reply:
Oh I get that, but we can also test that what would happen if we didn't surpress these exceptions.
eg:
- test that the entity doesn't go to unavailable
- test there is no reauth flow started
- test that we don't have an exception in the log (which we might have if we didn't surpress these.

Also, WebOsTvPairError is already caught by the try catch, is it still needed to surpress them?

Reply:
`WebOsTvPairError` should not be suppressed, I will test and fix as a bugfix